melodic/distribution.yaml

```diff
@@ -2138,6 +2138,33 @@ repositories:
         release: release/melodic/{package}/{version}
       url: https://github.com/pal-gbp/ddynamic_reconfigure_python-release.git
       version: 0.0.1-0
+  denso_robot_ros:
+    doc:
+      type: git
+      url: https://github.com/DENSORobot/denso_robot_ros.git
```

Comment:
There's no LICENSE file at the top-level of the repository.  While it isn't technically required, it is a good idea to have one at the top for people looking at your repository.

My suggestion is to add a LICENSE file with the MIT license in it.  Just put a comment here once you've done that.  No need to do another bloom release afterwards; it won't change the state of the release artifacts.

Reply:
Thank you for the suggestion. I've done it.

Reply:
I'm not seeing the added file. Did you add it to the source repo? I'm looking here https://github.com/DENSORobot/denso_robot_ros and there's no LICENSE file. I only see a `License.txt` in `bcap_core` dated 3 years ago with the MIT license, is that by chance the file added? I think the comment above means adding a file named LICENSE to the root of the repo.

Reply:
I added to [melodic-devel](https://github.com/DENSORobot/denso_robot_ros/tree/melodic-devel) branch. I'll merge master branch after this merge.